apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -568,9 +572,8 @@ const ApplicationDialogView: React.FC<Props> = ({
               className={inputClassName}
               placeholder={t('Enter the maximum number of attachments') || ''}
               onChange={(e) => {
-                const value = e.target.value
-                  ? Number(e.target.value)
-                  : undefined;
+                const value =
+                  e.target.value !== '' ? Number(e.target.value) : undefined;
```

Comment:
Are you sure that we handle spaces and NaN cases?
![image](https://github.com/user-attachments/assets/c651d03d-0f62-42f5-9c8b-9755f7219da8)